site/src/patterns/message.md

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
This page seems identical to the component page - keep one and remove the other?

Reply:
I updated the pages. Will wanted to have a patterns page as well. Not sure if i need to adjust verbiage